contracts/ts/genMaciState.ts

```diff
@@ -141,6 +142,35 @@ export const genMaciStateFromContract = async (
     // eslint-disable-next-line no-await-in-loop
     const publishMessageLogs = await pollContract.queryFilter(pollContract.filters.PublishMessage(), i, toBlock);
 
+    const [
+      joinPollLogs,
+      // eslint-disable-next-line no-await-in-loop
+    ] = await Promise.all([pollContract.queryFilter(pollContract.filters.PollJoined(), i, toBlock)]);
```

Comment:
No need to use Promise.all for only one async call

Reply:
https://github.com/0x3327/maci/commit/4c5c7d7fc3b256ec10690ec7da6077597d2f21d2